pyproject.toml

```diff
@@ -19,4 +19,5 @@ known-first-party = ["transformers"]
 [tool.pytest.ini_options]
 markers = [
     "run_slow",
+    "prequantized_model_test",
```

Comment:
Can we remove the `run_slow`?

Reply:
instead of run_slow, made it quant_test. Needed a marker for nightly test